src/sound_game.cpp

```diff
@@ -397,6 +399,11 @@ void handleLevelMusic()
 			activeminotaur = true;
 			break;
 		}
+		else if ( entity->sprite == 646 || entity->sprite == 650 )     // magister body
```

Comment:
Could we transition this whole section to use map.creatures and check for races instead of sprites? That would be way more efficient.